### PR TITLE
Fix callback hook for LifecycleLogger. Update specs to handle message…

### PR DESCRIPTION
**Pull request by mykola-kyryk**
**Feature:**
Fix callback hook for LifecycleLogger. Update specs to handle messages with guids

**Ticket:**
https://app.asana.com/0/31181193057789/31453749715074

**Submitter Checklist:**
- [x] CI Passing
- [x] PR Linked to Asana Ticket
- [x] Hound Appeased Acceptably
- [x] No Code Climate Issues
- [x] Class, Module, and public method comments up to spec

**Reviewer Checklist:**
- [ ] New Code Tested Effectively
- [ ] Code organization meets SOP standards
- [ ] Does this code fullfill the ticket?

**Only Then:**
- [ ] Move ticket in Asana

### CODE REVIEW

File: lib/dispatch-rider/logging/lifecycle_logger.rb [hunk 1]
```diff
@@ -11,8 +11,8 @@ def log_got_stop(reason, message)
         end
 
         def wrap_handling(message)
-          log_start(message)
           start_time = Time.now
+          log_start(message)
```
Comment (bramski) [inline lib/dispatch-rider/logging/lifecycle_logger.rb]: ?

**Comment by bramski** [comment]
Hmmm.  I don't quite understand what this is fixing.

File: lib/dispatch-rider/logging/lifecycle_logger.rb (hunk 1 above)
Comment (mykola-kyryk) [inline lib/dispatch-rider/logging/lifecycle_logger.rb]: if `log_start` fails, the `start_time` won't get initialized, but we are using it in an `ensure` block.

**Comment by piotrb** [comment]
The logging here isn't actually tested it seems .. that's part of why this needed fixing in the first place .. this doesn't seem to be adding them

**Comment by nicosuria** [comment]
What's blocking this PR?

**Pull request merged**